### PR TITLE
Heroku

### PR DESCRIPTION
**Pull request by alir128**
Once merged on master these changes will switch our toonin client and backend to heroku. The old backend without distributed streaming is currently hosted on heroku already. Master will get auto deployed to heroku due to our circle ci config.

### CODE REVIEW

**Comment by Lakshya2610** [comment]
@alir128 Give me the public DNS for the server after merging so I can update the A record for the domain

File: client/src/main.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import io from "socket.io-client";
 Vue.config.productionTip = false;
 
 // ATTN: Uncomment accordingly for local/remote dev
-const ENDPOINT = "https://www.toonin.ml:8443/";
+const ENDPOINT = "https://www.toonin.ml:443/";
```
Comment (Lakshya2610) [inline client/src/main.js]: Shouldn't this still be 8443. This is the Endpoint to the signalling server

Comment (alir128) [reply client/src/main.js]: everything is on 443 now. It's one node server.

Comment (Lakshya2610) [reply client/src/main.js]: Makes sense. Can you setup temporary redirect from 8443 to 443 for backend requests until we get the updated extension on webstore?

**Pull request merged**